### PR TITLE
Add hard-coded typography mixins

### PR DESCRIPTION
**Pull request by peterszerzo**
This PR adds hard-coded typography mixins, as in the example below. When using these mixins instead of the typography helper accepting a string argument, the sass compiler throws an error when the client mistypes the mixin name. Currently, the helper mixin throws a warning that may go silent in certain build setups and wouldn't crash continuous deployment processes - potentially leading to undesirable styling outcomes while using the library.

```scss
// Previously
h1 {
  @include typography(hero);
}

// With this PR
h1 {
  @include hero;
}
```

Backwards-compatibility is maintained, deprecation warning added for the generic typography mixin in sassdoc.

### CODE REVIEW

File: docs/docs/components/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "airtame-gooey",
-  "version": "1.3.0",
+  "version": "1.4.0",
```
Comment (mmellado) [inline docs/docs/components/package.json]: Please remove the bumping, this will be part of a separate commit together with a changelog

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "airtame-gooey",
-  "version": "1.3.0",
+  "version": "1.4.0",
```
Comment (mmellado) [inline package.json]: Please remove the bumping. This will be part of a separate commit together with the appropriate tags and changelog updates

Comment (peterszerzo) [inline package.json]: Will do :).

**Comment by atiti** [comment]
I guess these commits should be squashed together and resubmitted?
